test_tar_changes.py

```diff
@@ -41,13 +42,15 @@ def distgitdir(tmpdir):
 
 
 def test_commit_distgit_amend(distgitdir):
-    suffix = 'GitLab-User: developer@example.com'
+    # Test if commit_distgit_amend(suffix) returns the correct formatted git commit msg
+    # suffix = tarchanges.check_gitlabuser()
+    suffix = "GitLab-User: developer@example.com"
     rng = 'HEAD~..HEAD'
     with distgitdir.as_cwd():
         tarchanges.commit_distgit_amend(suffix)
         message = git('log', '--format=%s%n%n%b', rng, log_cmd=False)
-    expected = """\
+    matched = message.startswith("""\
```

Comment:
It's unclear to me why we need to change to `.startswith()` here, instead of matching the whole string.

Can we go back to the simple string matching?